### ISSUE TITLE
[bug]: `zinit creinstall -Q ...` or `-q` is not quiet

### ISSUE THREAD

**Issue by PyroDevil** [zdharma-continuum/zinit#518]
### What happened?

The documentation states that the `-q` or `-Q` parameter for `zinit creinstall` is quiet, but it prints `==> Installed 1 completion`.

This is annoying because I have put this command into the `zshrc` and thus it gets printed on every new shell.

### Steps to reproduce

1. Use zinit version `v3.11.0`
2. put some completion script into a directory, for instance `rustup completions zsh cargo > ".config/zsh/completions/_cargo"`
3. execute `zinit creinstall -Q ~/.config/zsh/completions` and expect no output.

### Relevant output

```shell
❯ zinit creinstall ~/.config/zsh/completions
Symlinking completion _cargo to completions directory.
Forgetting commands completed by `_cargo': cargo
==> Installed 1 completion
==> Added 1 completion to $INSTALLED_COMPS array
Initializing completion (compinit)…
❯ zinit creinstall -q ~/.config/zsh/completions
==> Installed 1 completion
❯ zinit creinstall -Q ~/.config/zsh/completions
==> Installed 1 completion
```

### Operating System & Version

OS: linux-gnu | Vendor: pc | Machine: x86_64 | CPU: x86_64 | Processor: unknown | Hardware: x86_64

### Zsh version

zsh 5.9 (x86_64-pc-linux-gnu)

### Terminal emulator

xterm-256color

### If using WSL on Windows, which version of WSL

None

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by vladdoster** [comment]
@PyroDevil,

Will fix this soon.

In the meantime, to silence it, append the following snippet:

```zsh
2>&1 > /dev/null
```

Example:

<img width="682" alt="Screenshot 2023-05-14 at 00 52 25" src="https://github.com/zdharma-continuum/zinit/assets/10052309/7b9b8be7-33d9-46b9-8d6a-141a3c65e840">

**Comment by vladdoster** [comment]
@PyroDevil,

If you have time, please test the patch.

```
zinit self-update
cd "$ZINIT[BIN_DIR]"
git pull --all
git checkout origin fix/creinstall-command
cd -
exec zsh -il
```

**Comment by vladdoster** [comment]
Yes, the `--quiet` flag is not entirely silent, which is by design. 



<img width="1344" alt="Screenshot 2023-05-14 at 11 24 42" src="https://github.com/zdharma-continuum/zinit/assets/10052309/c69c5957-71be-4acb-bbe5-15465220ac8a">

**Comment by vladdoster** [comment]
@PyroDevil,

Could you elaborate on why you are running `creinstall` in your `zshrc`?

**Comment by alichtman** [comment]
@vladdoster with `fix/creinstall-command`, I get the following error printed at the top of my terminal:

```
No such (plugin or snippet): -Q.
```

Silencing by piping output to `/dev/null` works (but is messy).

The reason I'm running `creinstall` is to get a custom completion I'm adding to be recognized:

```
rustup completions zsh cargo > $ZINIT_HOME/completions/_cargo

# Install my custom completions
zinit creinstall -Q $ZDOTDIR/completions
```

Is there a better way to do this?

**Comment by PyroDevil** [comment]
@vladdoster 

Sorry for the delay. I tested `fix/creinstall` `creinstall -q` or `creinstall -Q` prints `==> Skipped re-installing 1 completion`.

So that is not fixed there.

The reason I use `creinstall` in `.zshrc` is the same as @alichtman.

**Comment by Turiok** [comment]
I have the same problem.
I'm using poetry and kubectl completion. I have to creinstall to add theses completions
